main/guides/zoe/index.md

```diff
@@ -1,24 +1,38 @@
 # Zoe Overview
 
-## What is Zoe?
+The Zoe service and smart contract API support credibly trading assets with reduced risk.
 
-Zoe is the heart of Agoric's smart contract framework. Use Zoe to:
-
-- **Run your code on-chain**
-- **Mint new digital assets**
-- **Credibly trade assets**
-
-## Why Use Zoe?
-
-- **Zoe is Safer for Users:** A Zoe offer guarantees that you get either
+- **Zoe is Safer for Users:** Zoe guarantees that when you make an offer, you get either
```

Comment:
Would it make sense to change the phrasing from "you either get what you wanted or ..." to "you get either what you told the contract you wanted or ..."?  Do you think it's obvious that when we say "what you wanted", it's only the things you can express to the contract, and not the pony you also had in mind?

Reply:
at least "what you said you wanted" is worth changing, yes.

I think "told the contract" is reasonably implied by context; in particular, "Zoe service and smart contract API" in the sentence above.